charts/fluid/fluid/templates/csi/daemonset.yaml

```diff
@@ -33,75 +33,83 @@ spec:
                 values:
                 - virtual-kubelet
       containers:
-        - name: node-driver-registrar
-          image: "{{ .Values.csi.registrar.image }}"
-          args:
-            - --v=5
-            - --csi-address={{ .Values.csi.kubelet.rootDir }}/csi-plugins/fuse.csi.fluid.io/csi.sock
-            - --kubelet-registration-path={{ .Values.csi.kubelet.rootDir }}/csi-plugins/fuse.csi.fluid.io/csi.sock
-          env:
-            - name: KUBE_NODE_NAME
-              valueFrom:
-                fieldRef:
-                  fieldPath: spec.nodeName
-          volumeMounts:
-            - name: kubelet-dir
-              mountPath: {{ .Values.csi.kubelet.rootDir }}
-              mountPropagation: "HostToContainer"
-            - name: registration-dir
-              mountPath: /registration
-        - name: plugins
-          lifecycle:
-            preStop:
-              exec:
-                command: ["/bin/sh", "-c", "rm /registration/fuse.csi.fluid.io-reg.sock"]
-          securityContext:
-            privileged: true
-            runAsUser: 0
-            capabilities:
-              add: ["SYS_ADMIN"]
-            allowPrivilegeEscalation: true
-          image: "{{ .Values.csi.plugins.image }}"
-          command: ["/usr/local/bin/entrypoint.sh"]
-          args:
-            - "--nodeid=$(NODE_ID)"
-            - "--endpoint=$(CSI_ENDPOINT)"
-            - --v=5
-            {{- if .Values.csi.featureGates }}
-            - --feature-gates={{ .Values.csi.featureGates }}
-            {{- end }}
-            - "--pprof-addr=:6060"
-          env:
-            - name: NODE_ID
-              valueFrom:
-                fieldRef:
-                  fieldPath: spec.nodeName
-            - name: NODE_IP
-              valueFrom:
-                fieldRef:
-                  fieldPath: status.hostIP
-            {{- if .Values.runtime.mountRoot }}
-            - name: MOUNT_ROOT
-              value: {{ .Values.runtime.mountRoot | quote }}
-            {{- end }}
-            - name: ALLOW_PATCH_STALE_NODE
-              value: "true"
-            - name: KUBELET_ROOTDIR
-              value: {{ .Values.csi.kubelet.rootDir }}
-            - name: CSI_ENDPOINT
-              value: unix://{{ .Values.csi.kubelet.rootDir }}/csi-plugins/fuse.csi.fluid.io/csi.sock
-          imagePullPolicy: "IfNotPresent"
-          volumeMounts:
-            - name: plugin-dir
-              mountPath: /plugin
-            - name: kubelet-dir
-              mountPath: {{ .Values.csi.kubelet.rootDir }}
-              mountPropagation: "Bidirectional"
-            - name: fluid-src-dir
-              mountPath: {{ .Values.runtime.mountRoot | quote }}
-              mountPropagation: "Bidirectional"
-            - name: registration-dir
-              mountPath: /registration
+      - name: node-driver-registrar
+        image: "{{ .Values.csi.registrar.image }}"
+        {{- with .Values.image.imagePullSecrets }}
+        imagePullSecrets:
```

Comment:
`imagePullSecrets` should be in the same indentation level of `containers` instead of `containers[*].image`. Refering to the following example: https://kubernetes.io/docs/concepts/containers/images/#referring-to-an-imagepullsecrets-on-a-pod

Reply:
Thanks for your prompt, it has been fixed